// models/Donation.js
import mongoose from "mongoose";

const DonationSchema = new mongoose.Schema({

    email: {
        type: String,
        lowercase: true,
        trim: true,
        // required: true,
    },

    contactno: {
        type: Number,
        // required:true
    },

    addresstocollect: {
        type: String,
        // required:true
    },

    donationtype: {
        type: String,
        // required:true
    },

    condition: {
        type: String,

    },

    amount: {
        type: Number,
    },

    quantity: {
        type: String,
        // required:true
    },

    timeofcooking: {
        type: Date,
    },

    Status: {
        type: String,
        // required:true,
        default: "Pending"
    },

    addresstodonate: {
        type: String,
        // required:true,
    },

    message: {
        type: String
    },

    agentemail: {
        type: String,
        default: ''
    }

})

const DonationModel = mongoose.model("Donation", DonationSchema)

export { DonationModel as Donation }
